bundles/org.openhab.binding.pushover/src/main/java/org/openhab/binding/pushover/internal/connection/PushoverMessageBuilder.java

```diff
@@ -224,13 +235,22 @@ public ContentProvider build() throws CommunicationException {
             }
         }
 
+        if (!ttl.isZero()) {
+            if (priority == EMERGENCY_PRIORITY) {
+                logger.warn("TTL value of {} ignored for emergency priority.", ttl);
+            }
+            body.addFieldPart(MESSAGE_KEY_TTL, new StringContentProvider(String.valueOf(ttl.getSeconds())), null);
```

Comment:
If you really want to ignore it, a "else" is missing.

Reply:
Yes, it looks a little strange. I actually don't really want to ignore it, but according to the [API documentation](https://pushover.net/api#ttl):
> The ttl parameter is ignored for messages with a priority value of 2. The ttl value must be a positive number of seconds, and is counted from the time the message is received by our API.

So it's more like a warning that it will probably be ignored by the service. The only possibility to get into this situation is by using the full thing action method `sendMessage` providing all parameters explicitly, including priority and ttl.

Perhaps rephrasing to "TTL value of {} **will be** ignored for emergency priority." would be slightly more explicit, WDYT?

I didn't want to implement (hard code) these details from the API, but on the other hand thought a warning could be helpful.

Reply:
> Perhaps rephrasing to "TTL value of {} will be ignored for emergency priority." would be slightly more explicit, WDYT?

Yes, absolutely, it will be clearer.